client/lib/plans/personal-plan.js

```diff
@@ -0,0 +1,36 @@
+import { translate } from 'i18n-calypso';
+
+import { PLAN_PERSONAL } from './constants';
+
+export default {
+	product_id: 1009,
+	product_name: translate( 'WordPress.com Personal' ),
```

Comment:
I'm not sure how easy this would appear for translators, we would likely benefit from including some context

```js
translate( 'WordPress.com Personal', 'noun: name of Personal Plan' )
```